### PR TITLE
fix: cordel-viewer

### PR DESCRIPTION
**Pull request by samuelclerod**
# Pull Request Title

short description

what was changed?
- fix: ecordel viewer component

### CODE REVIEW

**Review by itsmemarioss — CHANGES_REQUESTED** [review]
o que acha de adicionarmos testes para esses comportamentos?
- toParagraphs
- toLines
- getSourceLink

**Comment by samuelclerod** [comment]
> o que acha de adicionarmos testes para esses comportamentos?
> 
> * toParagraphs
> * toLines
> * getSourceLink

Acho que vale a pena sim!

File: .github/workflows/ci_review.yml [hunk 1]
```diff
@@ -22,6 +22,8 @@ jobs:
         run: |
           npm install -g yarn
           yarn install --frozen-lockfile
+      - name: Run tests
+        run: yarn test
```
Comment (itsmemarioss) [inline .github/workflows/ci_review.yml:26]: Dessa forma os testes não são executados com watcher com interactive CLI? O runner das actions faz set da `CI`?

Comment (itsmemarioss) [reply .github/workflows/ci_review.yml:26]: mystery solved
https://docs.github.com/en/actions/learn-github-actions/environment-variables#default-environment-variables

**Review by itsmemarioss — APPROVED** [review]

**Pull request merged**